cmd/kanikoExecute.go

```diff
@@ -98,6 +99,10 @@ func runKanikoExecute(config *kanikoExecuteOptions, telemetryData *telemetry.Cus
 		if err != nil {
 			return errors.Wrapf(err, "failed to read file '%v'", config.DockerConfigJSON)
 		}
+	} else if len(config.ContainerRegistryUser) > 0 &&
```

Comment:
How about checking for `dockerCredentialsId` and `registryUrl`?

Reply:
The registry url is checked for two lines below. I'm not sure if the `dockerCredentialsId` is available in this go code. I believe the credentialsIds are Jenkins (i.e. groovy) specific.

Reply:
Then, you can add this credential '`dockerCredentialsId`' in `kanikoExecute.groovy` file, then it must be available in the go code right?

Reply:
Oh, you are right, I forgot that part, thanks!